hologram_dashboard/extension/src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -1,14 +1,23 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { DashboardComponent } from './dashboard.component';
+import { HttpClientModule } from '@angular/common/http';
 
 describe('DashboardComponent', () => {
   let component: DashboardComponent;
   let fixture: ComponentFixture<DashboardComponent>;
+  const mockJSONInput = '{"SPAM":[["5/19/2020",true],["5/18/2020",false]]}';
+  let sourceTypeElement: HTMLElement;
+  let firstDateElement: HTMLElement;
+  let secondDateElement: HTMLElement;
+  let dashboard: HTMLElement;
+  let input : JSON;
```

Comment:
Per Google typescript style guide, Use const by default, unless a variable needs to be reassigned. Ditto for other places.

Reply:
can't use const because they are initialized at a different time, if we were to initialize them then it'll be out of scope.